components/Ripple/tests/snapshot/MDCRippleViewSnapshotTests.m

```diff
@@ -83,4 +98,57 @@ - (void)testMaximumRadius {
   [self generateSnapshotAndVerifyForView:self.view];
 }
 
+- (void)testRippleColorRespondsToDynamicColorBeforeRippleBegan {
+#if defined(__IPHONE_13_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0)
+  if (@available(iOS 13.0, *)) {
+    // Given
+    UIColor *darkModeColor = UIColor.redColor;
+    UIColor *dynamicColor =
```

Comment:
Tip: Can these colors be put into properties or static functions?  Doing so would improve code readability for me.